grommet/grommet
# Accordion test warnings/errors fix

#### What does this PR do?
Removes the warnings and errors from `Accordion-test.js`. Does with by replacing `setTimeout()` with mocked timers from jest and also confirming console warnings. 
#### Where should the reviewer start?
`grommet/src/js/components/Accordion/Accordion-test.js`
#### What testing has been done on this PR?
I ran `yarn test Accordion-test` to confirm that test coverage did not change and that the warnings were properly suppressed
#### How should this be manually tested?
Run `yarn test Accordion-test` to confirm that the tests still work and the warnings/errors were resolved
#### What are the relevant issues?
#4177 
#### Screenshots (if appropriate)
<img width="751" alt="Screen Shot 2020-06-17 at 2 11 15 PM" src="https://user-images.githubusercontent.com/49867133/84933817-73136f00-b0a4-11ea-9c0c-cb42e8cd6456.png">

#### Do the grommet docs need to be updated?
No
#### Should this PR be mentioned in the release notes?
No
#### Is this change backwards compatible or is it a breaking change?
Backwards compatible
#### Note:
I was not able to use `expect(warnSpy).toHaveBeenCalledWith()` because due to the warning being a multi-line string I could not get the expected value passed into `toHaveBeenCalledWith()` to match the actual value
<img width="587" alt="Screen Shot 2020-06-17 at 2 19 26 PM" src="https://user-images.githubusercontent.com/49867133/84934662-a7d3f600-b0a5-11ea-9de1-7548558063be.png">
